### PR TITLE
docs: How to get console output on Windows

### PR DESCRIPTION
**Pull request by pbodnar**
See discussion at #349 and others -> a piece of documentation should come handy.

### CODE REVIEW

**Comment by hluk** [comment]
Awesome! Thanks.

File: docs/command-line.rst [hunk 1]
```diff
@@ -5,6 +5,14 @@ Tabs, items, clipboard and configuration can be changed through command
 line interface. Run command ``copyq help`` to see complete list of
 commands and their description.
 
+.. warning::
+
+    On Windows, you won't see any output from running ``copyq`` on
+    command line. Luckily there are various workarounds, like:
+    
+    * Run it from Git Bash, Cygwin or similar.
```
Comment (Chealer) [inline docs/command-line.rst:13]: This is rather misleading. For example, running Bash in Windows Terminal still fails. I suggest referring to [the known issue](https://copyq.readthedocs.io/en/latest/known-issues.html#on-windows-copyq-does-not-print-anything-on-console) instead.

Comment (pbodnar) [reply docs/command-line.rst:13]: Well, this was nearly 6 years ago... :D

> For example, running Bash in Windows Terminal still fails.

Yeah, at the time of writing, I guess we haven't realized it actually depends more on the type of _terminal_ used. There is a continuation on this topic at #349.

> I suggest referring to [the known issue](https://copyq.readthedocs.io/en/latest/known-issues.html#on-windows-copyq-does-not-print-anything-on-console) instead.

This was already been done - see 5a792fe.

Comment (Chealer) [reply docs/command-line.rst:13]: Oh, sorry and thank you Petr and @hluk

**Pull request merged**